core/core.go

```diff
@@ -210,6 +248,46 @@ func (n *IpfsNode) OnlineMode() bool {
 	return n.onlineMode
 }
 
+func (n *IpfsNode) Bootstrap(ctx context.Context, peers []peer.PeerInfo) error {
+	if n.DHT != nil {
```

Comment:
It originally used the `bootstrap` method, but was causing tests to hang. I'll put a TODO comment in the function as a reminder to address before merging.
```
func bootstrap(ctx context.Context, h host.Host, r *dht.IpfsDHT, ps peer.Peerstore, bootstrapPeers []peer.PeerInfo) error
```